Truck.js

```diff
@@ -1,6 +1,27 @@
 var Car = require('./Car.js');
 
 //Implement your Truck here
-function Truck() {};
+
+function Truck(make, model, year, color, seats, capacity, cargo) {
+	Car.call(this, make, model, year, color);
+	this.seats = 3;
+	this.capacity = capacity || 0;
+	this.cargo = cargo || 0;
+};
+
+Truck.prototype = Object.create(Car.prototype);
+Truck.prototype.constructor = Truck;
+
+Truck.prototype.load = function(pounds) {
+	this.cargo = pounds;
```

Comment:
Here, ```cargo``` is separate from the amount that we want to load (```pounds```). For instance, let's say we have 200 pounds of cargo already in the truck, and we want to add another 200 pounds. We would need to set ```this.cargo += pounds```.